app/lib/actions/katello/content/create.rb

```diff
@@ -0,0 +1,18 @@
+module Actions
+  module Katello
+    module Content
+      class Create < Actions::Base
+        def run
+          content = ::Katello::Content.new(name: input[:name],
+                                              cp_content_id: input[:cp_content_id],
+                                              content_type: input[:content_type],
+                                              label: input[:label],
+                                              content_url: input[:content_url])
+          success = content.save
```

Comment:
We typically call .save! here to bubble the error up as an exception and pause the action.  Here the results would be saved, but the action would continue (even though an error occured)